packages/@angular/cli/utilities/find-up.ts

```diff
@@ -16,7 +18,7 @@ export function findUp(names: string | string[], from: string, stopOnPackageJson
       }
     }
 
-    if (stopOnPackageJson) {
+    if (!isInHomeDir && stopOnPackageJson) {
```

Comment:
Maybe we should separate these checks. `find-up` itself shouldn't care about home dirs.